crystal/distribution.yaml

```diff
@@ -834,15 +834,15 @@ repositories:
       url: https://github.com/micro-ROS/micro-ROS-Agent.git
       version: v0.0.1
     status: developed
-  micro-xrce-dds-agent:
+  microxrcedds_agent:
```

Comment:
It looks like [CI is failing](https://travis-ci.org/ros/rosdistro/jobs/557799219) with 

```
======================================================================
ERROR: test.test_build_caches.test_build_caches
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python2.7.14/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/travis/build/ros/rosdistro/test/test_build_caches.py", line 57, in test_build_caches
    raise RuntimeError('\n'.join(errors))
RuntimeError: crystal: missing package.xml file for package "microxrcedds_agent"
```

Maybe this is because the [package.xml still has the previous packages name](https://github.com/micro-ROS/microxrcedds_agent-release/blob/release/crystal/micro-xrce-dds-agent/1.0.3-1/package.xml)?